modules/error_log/utils.py

```diff
@@ -0,0 +1,28 @@
+import discord
+import discord.utils
+import os
+import csv
+from datetime import datetime
+
+def get_discord_obj(iterable, label: str):
+	def get_id(label: str):
+		"""gets the id of an object that has the given label in the CSV file"""
+		with open("ids.csv") as csv_file:
+			csv_reader = csv.reader(csv_file, delimiter=",")
+			for row in csv_reader:
+				if row[0] == label:
+					return int(row[1])
+			return None
+	return discord.utils.get(iterable, id=get_id(label))
+
+def get_logs(filename: str):
```

Comment:
Should probably be in the error logging class I mentioned before. Same applies to some other functions in this file